docs/migrating_to_20_1.md

```diff
@@ -1,3 +1,101 @@
 # Migrating to Bento v20.1
 
-TODO
+## Minio to Garage Migration
+
+### Prerequisites
+Before starting the migration, ensure you have:
+
+> [!IMPORTANT]
+> If using MinIO, **DO NOT** check out the `v20.1` tag or release branch until you've downloaded all data from MinIO for migration to Garage.
+
+- Access to both Minio instance
+- [`s3cmd`](https://github.com/s3tools/s3cmd) installed and configured
+- Sufficient local disk space for the backup
+
+### Step 1: Back up data from Minio
+
+```bash
+# Sync data from Minio to local backup
+s3cmd -c ~/.s3cfg-minio-local sync s3://drop-box/ ./drop-box-backup/
+s3cmd -c ~/.s3cfg-minio-local sync s3://drs/ ./drs-backup/
+```
+
+For example, your `.s3cfg-minio-local` might look something like this:
+```ini
+[default]
+host_base = minio.bentov2.local  
+host_bucket = minio.bentov2.local  
+use_https = True  
+
+# For dev self-signed certs only  
+check_ssl_certificate = False  
+
+# Credentials  
+# Re-use ones in your local.env, or create new ones in the minio UI at minio.bentov2.local  
+access_key = <REDACTED>  
+secret_key = <REDACTED>  
+```
+
+**Notes**
+- Use `--dry-run` flag to preview sync operations before executing
+- Add `--delete-removed` if you want to mirror deletions
+- Consider using `--skip-existing` for incremental syncs
+
+### Step 2: Setting up Garage
+
+Now you can switch to Bento - v20.1; Follow the steps listed [here](./garage.md) to setup Garage.
+
+### Step 3: Upload data to Garage
+
+```bash
+# Sync local backup to Garage
+s3cmd -c ~/.s3cfg-garage-local sync ./drop-box-backup/ s3://drop-box/
+s3cmd -c ~/.s3cfg-garage-local sync ./drs-backup/ s3://drs/
+```
+
+S3cfg template for Garage: 
+
+```ini
+# ~/.s3cfg-garage-local
+[default]
+access_key = <GARAGE_ACCESS_KEY>
+secret_key = <GARAGE_SECRET_KEY>
+host_base = garage.bentov2.local
+host_bucket = garage.bentov2.local/%(bucket)s
```

Comment:
only worked for me without `/%(bucket)s` (see https://garagehq.deuxfleurs.fr/documentation/connect/cli/#s3cmd )